app/models/manageiq/providers/nsxt/manager_mixin.rb

```diff
@@ -86,6 +86,25 @@ def params_for_create
                     :validate     => [{:type => "required"}],
                     :initialValue => 443,
                   },
+                  {
+                    :component    => "select",
+                    :id           => "endpoints.default.api_version",
+                    :name         => "endpoints.default.api_version",
+                    :label        => _("API Version"),
+                    :isRequired   => true,
+                    :validate     => [{:type => "required"}],
+                    :initialValue => "GM_V1",
+                    :options      => [
+                      {
+                        :label => _("Global Manager - Version 1"),
+                        :value => "GM_V1"
+                      },
+                      {
+                        :label => _("Local Manager - Version 1"),
+                        :value => "LM_V1"
+                      }
```

Comment:
It feels like we're overriding the `Endpoint#api_version` with the api_version plus if it is global or local.

We could set the `Endpoint#path` attribute with similar logic to what you have in the nsxt client below

Reply:
Change from api_version to path as Manage Role selection